### ISSUE TITLE
Feature suggestion

### ISSUE THREAD

**Issue by MSAbaddon** [jopeek/fvtt-loot-sheet-npc-pf2e#3]
Allow more than one table to be selected, including separate formulas for rolls and quantities. This way a merchant can have both some single permanent items such as weapons and armor as well as a decent amount of potions and scrolls in larger quantity.

**Comment by jopeek** [comment]
Just to clarify, you can change tables and formula and just keep adding to the sheet, but your suggestion is noted anyway.

**Comment by MSAbaddon** [comment]
Yes, I know, would be great though if it's possible to fill up a merchant with just one refresh click :-)
Thanks for the quick reply.
